lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -2,42 +2,43 @@ const { evaluators } = require('arsenal').policies;
 const constants = require('../../../../constants');
 
 const actionMap = {
-    's3:AbortMultipartUpload': 'multipartDelete',
-    's3:DeleteBucket': 'bucketDelete',
-    's3:DeleteBucketPolicy': 'bucketDeletePolicy',
-    's3:DeleteBucketWebsite': 'bucketDeleteWebsite',
-    's3:DeleteObject': 'objectDelete',
-    's3:DeleteObjectTagging': 'objectDeleteTagging',
-    's3:GetBucketAcl': 'bucketGetACL',
-    's3:GetBucketCORS': 'bucketGetCors',
-    's3:GetBucketLocation': 'bucketGetLocation',
-    's3:GetBucketObjectLockConfiguration': 'bucketGetObjectLock',
-    's3:GetBucketPolicy': 'bucketGetPolicy',
-    's3:GetBucketVersioning': 'bucketGetVersioning',
-    's3:GetBucketWebsite': 'bucketGetWebsite',
-    's3:GetLifecycleConfiguration': 'bucketGetLifecycle',
-    's3:GetObject': 'objectGet',
-    's3:GetObjectAcl': 'objectGetACL',
-    's3:GetObjectLegalHold': 'objectGetLegalHold',
-    's3:GetObjectRetention': 'objectGetRetention',
-    's3:GetObjectTagging': 'objectGetTagging',
-    's3:GetReplicationConfiguration': 'bucketGetReplication',
-    's3:ListBucket': 'bucketHead',
-    's3:ListBucketMultipartUploads': 'listMultipartUploads',
-    's3:ListMultipartUploadParts': 'listParts',
-    's3:PutBucketAcl': 'bucketPutACL',
-    's3:PutBucketCORS': 'bucketPutCors',
-    's3:PutBucketObjectLockConfiguration': 'bucketPutObjectLock',
-    's3:PutBucketPolicy': 'bucketPutPolicy',
-    's3:PutBucketVersioning': 'bucketPutVersioning',
-    's3:PutBucketWebsite': 'bucketPutWebsite',
-    's3:PutLifecycleConfiguration': 'bucketPutLifecycle',
-    's3:PutObject': 'objectPut',
-    's3:PutObjectAcl': 'objectPutACL',
-    's3:PutObjectTagging': 'objectPutTagging',
-    's3:PutObjectLegalHold': 'objectPutLegalHold',
-    's3:PutObjectRetention': 'objectPutRetention',
-    's3:PutReplicationConfiguration': 'bucketPutReplication',
+    multipartDelete: 's3:AbortMultipartUpload',
```

Comment:
With this it essentially becomes same as https://github.com/scality/Arsenal/blob/32752ac50454b24eeaf639065618df1fb18502fc/lib/policyEvaluator/RequestContext.js#L22
I am not asking for now as we are close to cc, please make this list in one place (arsenal) and import it here.